stdlib/Dates/src/types.jl

```diff
@@ -401,6 +401,12 @@ Base.eps(::Type{Time}) = Nanosecond(1)
 Base.eps(::T) where T <: TimeType = eps(T)::Period
 
 
+Base.zero(::Type{DateTime}) = Millisecond(0)
```

Comment:
why is zero of a `DateTime` a `Miilisecond`, but zero for a `Time` a `Nanosecond`? Can the answer to that be added as a comment?

Reply:
It returns x::T - x::T (added as a comment).

Reply:
Oh that's a good reason! thanks for adding the comment 👍